parlai/tasks/taskmaster2/README.md

```diff
@@ -1,5 +1,5 @@
 # Taskmaster 2
 
 Originally from the
-[Google Research Datasets](https://github.com/google-research-datasets/Taskmaster/blob/main/TM-2-2020/README.md).
+[Google Research Datasets](https://github.com/google-research-datasets/Taskmaster/blob/master/TM-2-2020/README.md).
```

Comment:
This is indeed the correct path, interestingly enough. Not sure exactly where this changed though hmm...

Reply:
Wild to see Google make the opposite push as us.

Reply:
Looking at the history of that file, I think the `master->main` change came in from a regex from when we did that. (No intentional moving from `main->master` on their part.